code/modules/vending/lobotomyarmband.dm

```diff
@@ -5,16 +5,16 @@
 	icon_deny = null
 	req_access = null //Shouldn't need access on lc13, can be changed
 	vend_reply = "Armband vended."
-	products = list(/obj/item/clothing/accessory/armband/lobotomy = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/command = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/training = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/info = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/safety = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/discipline = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/welfare = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/records = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/extraction = 4,
-					/obj/item/clothing/accessory/armband/lobotomy/architecture = 4)
+	products = list(/obj/item/clothing/accessory/armband/lobotomy = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/command = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/training = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/info = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/safety = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/discipline = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/welfare = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/records = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/extraction = 20,
+					/obj/item/clothing/accessory/armband/lobotomy/architecture = 20)
```

Comment:
Remove architecture armbands. Those are meant to be a "reward" for long time players.